### PR TITLE
fix(numeric-list): Fix numeric list color on dark-mode

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Before:**

image

image

**After:**

image
image

### CODE REVIEW

**Comment by priscilawebdev** [comment]
@vuluongj20 I've updated the list to have the color theme.black ` but I'm not sure if there is a better color

**Comment by priscilawebdev** [comment]
Also thought about using `theme.badge.warning.color `but then we have the same problem mentioned in the description

**Review by narsaynorath — APPROVED** [review]
Looks good to me, I checked out the widget builder, alerts forms, and AWS lambda integration form and the numbers are now visible in dark mode

File: static/app/components/list/index.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const List = styled(
   ${p =>
     typeof p.symbol === 'string' &&
     listSymbol[p.symbol] &&
-    getListSymbolStyle(p.theme, p.symbol, p.initialCounterValue)}
+    getListSymbolStyle(p.theme, p.symbol, p.initialCounterValue)};
```
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply static/app/components/list/index.tsx:60]: this was automatically added when i was saving, but I can removed

Comment (priscilawebdev) [inline static/app/components/list/index.tsx:60]:
```suggestion
    getListSymbolStyle(p.theme, p.symbol, p.initialCounterValue)}
```

**Pull request merged**